CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ## [Unreleased]
 
+## [0.20.0] - 2019-12-06
+### Added
+- Introduced `Tags` class unifying and simplifying access to SPEC tags
+- Added proper support for *crate* and *gem* archives
+- Added `--bugzilla-id` option to perform a rebase based on *Upstream Release Monitoring* bugzilla
+- Added option to define macros
```

Comment:
I think you should also add the specific option (`-D`) to stay consistent with the line above.

Reply:
But what about `--define`? :wink:

Reply:
But what about `--define`? :wink: